test/common/utils/formatNumber.test.ts

```diff
@@ -0,0 +1,89 @@
+import {assertEquals} from "@std/assert";
+import {describe, it} from "@std/testing/bdd";
+import {CommonLocales, formatNumber} from "../../../src/utils/format-number";
+
+describe("Correct Format Number Utils", () => {
+    // Use a set to store all common locales.
+    // It is useful to invalidate the test if a new locale is added and not tested.
+    const commonLocales: Set<string> = new Set(Object.values(CommonLocales));
+    it("should format number using default, the English (United States) locale", () => {
+        const expected: string = (1000).toLocaleString("en-US");
```

Comment:
I think it would be better if you write yourself the expected string, which is "1,000" in this case, or "1 000" in French for instance. With your way it's correct, but you call the method which is really called in the function. So if one day (could be never though), the toLocaleString changes, your tests won't fail and you'll have another string format.
Sorry, English is not my native language so if you wish more explanations, just ask ;)

Reply:
Yes, at first I wrote the number as expected (like 1000, 1 000, and so on), but later I change it 😄.

Can I change it easily, that's a good point. And don't worry about the language, I'm Italian, so I know how you feel 😄.

Reply:
@Zhykos Done! See: https://github.com/Zhykos/autophoto/pull/72/commits/521b75152eb5817b1fb5a5027c8975b8f175621d